html-project-Leek995/index.js

```diff
@@ -0,0 +1,65 @@
+$(document).ready(function (){
```

Comment:
Love the use of JQuery! Super cool that you know how to use that library! Looking forward to seeing what you can do with react!